test/metabase/driver/common/parameters/dates_test.clj

```diff
@@ -201,6 +213,36 @@
                  (params.dates/date-string->range s options))
               (format "%s with options %s should parse to %s" (pr-str s) (pr-str options) (pr-str expected))))))))
 
+(deftest relative-dates-with-starting-from-zero-must-match
+  (testing "relative dates need to behave the same way, offset or not."
+    (t/with-clock (t/mock-clock #t "2016-06-07T12:13:55Z")
```

Comment:
We actually have an `mt/with-clock` macro that can take a temporal object directly. It also adds `testing` context and checks to make sure you're not using it inside a `^:parallel` test. So I'd recommend using that instead of `t/with-clock` directly. Now that we have the capability of writing custom linters in Kondo we should probably go add this in there at some point

```suggestion
    (mt/with-clock #t "2016-06-07T12:13:55Z"
```

Reply:
Good point, I'll fix the other `(with-clock .. mock-clock ..` calls in there too.